script/DeployDutchV2.s.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.13;
+
+import "forge-std/console2.sol";
+import "forge-std/Script.sol";
+import {IPermit2} from "permit2/src/interfaces/IPermit2.sol";
+import {V2DutchOrderReactor} from "../src/reactors/V2DutchOrderReactor.sol";
+import {OrderQuoter} from "../src/lens/OrderQuoter.sol";
+import {DeployPermit2} from "../test/util/DeployPermit2.sol";
+
+struct V2DutchOrderDeployment {
+    IPermit2 permit2;
+    V2DutchOrderReactor reactor;
+    OrderQuoter quoter;
+}
+
+contract DeployDutchV2 is Script, DeployPermit2 {
+    address constant PERMIT2 = 0x000000000022D473030F116dDEE9F6B43aC78BA3;
+    address constant UNI_TIMELOCK = 0x1a9C8182C09F50C8318d769245beA52c32BE35BC;
+
+    function setUp() public {}
+
+    function run() public returns (V2DutchOrderDeployment memory deployment) {
+        vm.startBroadcast();
+        if (PERMIT2.code.length == 0) {
+            deployPermit2();
+        }
+
+        V2DutchOrderReactor reactor = new V2DutchOrderReactor{salt: 0x00}(IPermit2(PERMIT2), UNI_TIMELOCK);
+        console2.log("Reactor", address(reactor));
```

Comment:
```suggestion
        console2.log("Reactor: ", address(reactor));
```

Reply:
i assume you want a space like that so they dont get logged all stuck together?

Reply:
and same with quoter

Reply:
it gets logged like this currently
== Logs ==
  Reactor 0x3867393cC6EA7b0414C2c3e1D9fe7cEa987Fd066
  Quoter 0xAA6187C48096e093c37d2cF178B1e8534A6934f7

Reply:
console2 automatically adds a space between separate params

Reply:
oh thats lovely of them